### DISCUSSION TITLE
Invariant site add issue with write_vcf()

### DISCUSSION THREAD

**Discussion by romieuj** [tskit-dev/tskit#2433] [Q&A]
Hello,
First of all, thank you for having developed an extremely practical package to manipulate simulated or real DNA sequences. 

I contact you because I use Tree sequences files to manipulate simulated sequences with msprime and SLiM (msprime for populations split ==> SLiM add positively selected mutation ==> msprime add neutral mutation). I have a problem when I transform a tree sequence file into a vcf (with write_vcf() : https://tskit.dev/tskit/docs/stable/python-api.html#tskit.TreeSequence.write_vcf) I have systematically the addition of an invariant site at position 15000000 in the vcf , I would like to know if you could help me to find the origin of the problem ? 

Thank you in advance for your help.  
Have a nice day!

**Comment by jeromekelleher** [comment]
Hi @romieuj :wave:

Can you give us a bit more information please?  Does this happen with or without putting the nuetral mutations in msprime? Is there anything special about position 15000000 in your code? I don't think this will have much to do with VCF per se, it's just that you are happening to produce a site with no variation at this particular position via your simulations.

**Comment by romieuj** [comment]
Indeed, it is the position where I add the positively selected mutation in the genome of one of my populations. As I use mutation.convertToSubstitution=F in SLiM the mutation is kept in the tree sequence file even when it is fixed.  I'm sorry to have wasted your time, I should have thought about it ! Thanks anyway for making me realize my mistake. 
Have a nice evening.

**Reply by jeromekelleher** [reply]
Haha, no problems @romieuj, happy simulating!
